akka-remote/src/main/scala/akka/remote/RemoteActorRefProvider.scala

```diff
@@ -623,6 +622,7 @@ private[akka] class RemoteActorRefProvider(
   def quarantine(address: Address, uid: Option[Long], reason: String): Unit =
     transport.quarantine(address, uid, reason)
 
+  override private[akka] def addressString = local.addressString
```

Comment:
is that right? shouldn't it include the host and port, which come from RemoteActorRefProvder.getDefaultAddress?

care must be take to handle the case when this is accessed before the provider is `init`. Then I think `_internals` is `null`. Probably alright to use `local.addressString` before init.

Reply:
You are right, was too happy that I could eliminate some logic and put it only in local. Dang.